gui-lib/mred/private/wxme/editor-canvas.rkt

```diff
@@ -466,49 +515,56 @@
 
   (define/public (popup-for-editor b m) #f)
 
+  ;; Reduces an ordered set of scroll events into the next scroll
+  ;; position and then scrolls to it.
+  (define (process-scroll-events events)
```

Comment:
I'm not completely sure, but this looks like a field bound to a lambda? Maybe it should be a private method instead.

Reply:
I went ahead and merged `process-scroll-events` into `on-scroll-flush`. There wasn't much point in having them be separate.